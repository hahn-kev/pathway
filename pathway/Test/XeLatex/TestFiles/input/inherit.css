.ta
{
    font-family: "Arial";
	font-size: 20pt;
	font-style: italic;
	color: red;
}
.tb
{
	font-family: inherit;
	font-size: inherit;
	font-style: inherit;
    color: blue;
}
.tc
{
    font: 200% "Times New Roman", serif;
    color: green;
}


