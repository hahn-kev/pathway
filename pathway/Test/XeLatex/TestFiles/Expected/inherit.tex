\documentclass[a4paper]{article} 
\usepackage{changepage}
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}
\usepackage{calc}
\usepackage{lettrine}
\usepackage[margin=2cm,includeheadfoot]{geometry}

\begin{document} 
\pagestyle{plain} 
\font\tctbta="Times New Roman/I":color=008000 at 40pt
\font\tbta="Arial/I":color=0000ff at 20pt
\font\ta="Arial/I":color=ff0000 at 20pt

\pagestyle{fancy} 

\ta{T1 class }

\tbta{T2 class }

\tctbta{T3 class }

\tbta{T2 class }

\ta{T1 class }

\end{document}
